.golangci.yaml

```diff
@@ -11,3 +11,5 @@ linters:
     - gofmt
     - gosec
     - govet
+  disable:
+    - errcheck
```

Comment:
We intending to disable this?

Reply:
I added it intentionally because tests are failing on "you didn't check for a return error value" in the tests themselves

Reply:
https://github.com/replicatedhq/troubleshoot/actions/runs/11486710569/job/31969600112?pr=1624#step:5:403

Reply:
looks like because of this most recent change https://github.com/replicatedhq/troubleshoot/commit/402d1117458ee4e7bacf66d3d4662ad1b00389f0